src/css/index.scss

```diff
@@ -0,0 +1,418 @@
+      float: right;
+      line-height: 70px;
+      margin: 0 20px;
+      a {
+        text-decoration: none;
+        color: $primary_color;
+        padding: 10px 20px;
+        border: 1px solid $btn_border_color;
+      }
+      a:hover {
+        background: $btn_hover_color;
+      }
+    }
+    .media_unique {
+      max-width: 50%;
+      position: relative;
+      margin: 0;
+      right: 0;
+      float: right;
+      line-height: 70px;
+      margin: 0 30px;
+      a {
+        text-decoration: none;
+        color: $primary_color;
+        padding: 0 8px;
+        font-size: 20px;
+      }
+      a:hover {
+        color: $btn_hover_color;
+      }
+    }
+  }
+}
+
+.bottom {
+  .bottom-content {
+    width: 100%;
+    display: flex;
+    position: relative;
+    .intro {
+      max-width: 50%;
+      width: auto;
+      height: auto;
+      position: relative;
+      display: inline-block;
+      margin: 0 20px 20px;
+    }
+    .experience {
+      max-width: 33.3%;
+      width: auto;
+      height: auto;
+      position: relative;
+      display: inline-block;
+      margin: 0 20px;
+      a {
+        text-decoration: none;
+        color: $primary_color;
+      }
+    }
+    .education {
+      max-width: 33.3%;
+      width: auto;
+      height: auto;
+      position: relative;
+      display: inline-block;
+      margin: 0 20px;
+    }
+    .hobbies {
+      max-width: 15%;
+      width: 100px;
+      height: auto;
+      position: relative;
+      display: inline-block;
+      margin: 15px 20px 0;
+      cursor: pointer;
+      i {
+        font-size: 50px;
+        text-align: center;
+        margin: 0 20px;
+      }
+      p {
+        font-size: 20px;
+        text-align: center;
+      }
+    }
+  }
+}
+
+.dot-sections {
+  text-align: center;
+  margin-top: 20px;
+  .dot {
+    cursor: pointer;
+    height: 15px;
+    width: 15px;
+    margin: 0 2px;
+    background-color: #bbb;
+    border-radius: 50%;
+    display: inline-block;
+  }
+  .dot:hover {
+    background-color: $btn_hover_color;
+  }
+  .active {
+    background: $btn_hover_color;
+  }
+}
+
+//fade animation
+.fade {
+  -webkit-animation-name: fade;
+  -webkit-animation-duration: 1.5s;
+  animation-name: fade;
+  animation-duration: 1.5s;
+}
+
+@-webkit-keyframes fade {
+  from {
+    opacity: 0.4;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes fade {
+  from {
+    opacity: 0.4;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+.skills-section {
+  max-width: 1000px;
+  position: relative;
+  margin: auto;
+  color: $primary_color;
+  padding: 0 20px;
+  h2 {
+    color: $secondary_color;
+  }
+  .skill {
+    margin: 0 20px;
+    max-width: 33.3%;
+    position: relative;
+    width: auto;
+    height: auto;
+    display: inline-block;
+  }
+}
+
+.contacts-section {
+  max-width: 1000px;
+  margin: auto;
+  color: $primary_color;
+  padding: 0 20px;
+  position: relative;
+  display: flex;
+  margin-bottom: 30px;
+  .id-section {
+    max-width: 50%;
```

Comment:
I think the only thing you need with in this block is width: 50%;
Please remove max-width: 50%; position: relative; height: auto; display: inline-block;